src/components/ButtonClientMobile.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-env browser */
```

Comment:
In fact, you could get rid of the linter's alert by using `window.localStorage`.
I understand the needs, but it is not easy to understand that this comments is for `localStorage`.